src/common/test/test_debugger_communication_client.py

```diff
@@ -3,10 +3,25 @@
 from unittest import mock
 import socketio
 import threading
+import os
+import sys
 
-from adafruit_circuitplayground import express
-from common import debugger_communication_client
```

Comment:
Just curious, if we used relative paths would the tests use the correct library?